demo/rmm_plugin/README.rst

```diff
@@ -58,4 +58,20 @@ Since with RMM the memory pool is pre-allocated on a specific device, changing t
 device ordinal in XGBoost can result in memory error ``cudaErrorIllegalAddress``. Use the
 ``CUDA_VISIBLE_DEVICES`` environment variable instead of the ``device="cuda:1"`` parameter
 for selecting device. For distributed training, the distributed computing frameworks like
-``dask-cuda`` are responsible for device management.
\ No newline at end of file
+``dask-cuda`` are responsible for device management.
+
+************************
+Memory Over-Subscription
+************************
+
+.. warning::
+
+   This feature is still experimental and is under active development.
+
+The newer NVIDIA platforms like `Grace-Hopper
+<https://www.nvidia.com/en-us/data-center/grace-hopper-superchip/>`__ use `NVLink-C2C
+<https://www.nvidia.com/en-us/data-center/nvlink-c2c/>`__, which allows the CPU and GPU to
+have a coherent memory model. Users can use the `SamHeadroomMemoryResource` in the latest
+RMM to utilize system memory for storing data. This can help XGBoost utilize memory from
+the host for GPU computation, but it may reduce performance due to page migration
```

Comment:
Probably both slower CPU memory speed and page migration overhead.

Reply:
Done.